Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -2242,6 +2242,24 @@ public IntradayVwap VWAP(Symbol symbol)
             return intradayVwap;
         }
 
+        /// <summary>
+        /// Creates a new VolumeWeightedMovingAverage indicator for the symbol. The indicator will be automatically
+        /// updated on the given resolution.
+        /// </summary>
+        /// <param name="symbol">The symbol whose VWMA we want</param>
+        /// <param name="period">The smoothing period used to smooth the computed VWMA values</param>
+        /// <param name="resolution">The resolution</param>
+        /// <param name="selector">Selects a value from the BaseData to send into the indicator, if null defaults to casting the input value to a TradeBar</param>
+        /// <returns>A new AverageTrueRange indicator with the specified smoothing type and period</returns>
```

Comment:
- XML documentation seems off, copy/paste
- `InitializeIndicator` needs to receive the symbol too, please take a look at how it works 👍